kevm-pyk/src/kevm_pyk/kproj/evm-semantics/optimizations.md

```diff
@@ -191,7 +191,7 @@ module EVM-OPTIMIZATIONS
       ...
     </kevm>
     requires ( #if USEGAS #then Gverylow < SCHED > <=Gas GAVAIL #else true #fi )
-     andBool ( #sizeWordStack( chop( ( W0 +Int W1 ) ) : WS ) <=Int 1024 )
+     andBool ( #sizeWordStack ( WS ) <=Int 1022 )
```

Comment:
This doesn't look correct to me? It sohuld be 1023?

Reply:
I think that it is correct, given that the size of the word stack is reducing and `W0 : W1 : WS` has to be of max size 1024?